nipype/interfaces/ants/utils.py

```diff
@@ -1,11 +1,456 @@
-# -*- coding: utf-8 -*-
-"""ANTS Apply Transforms interface
-"""
-
+"""ANTs' utilities."""
 import os
+from ...utils.imagemanip import copy_header as _copy_header
+from ..base import traits, isdefined, TraitedSpec, File, Str, InputMultiPath
```

Comment:
`InputMultiPath` is now `InputMultiObject`.

```suggestion
from ..base import traits, isdefined, TraitedSpec, File, Str, InputMultiObject
```